src/esp/gfx/LightSetup.cpp

```diff
@@ -15,32 +15,63 @@ bool operator!=(const LightInfo& a, const LightInfo& b) {
   return !(a == b);
 }
 
-Magnum::Vector3 getLightPositionRelativeToCamera(
+Magnum::Vector4 getLightPositionRelativeToCamera(
     const LightInfo& light,
     const Magnum::Matrix4& transformationMatrix,
     const Magnum::Matrix4& cameraMatrix) {
+  ASSERT(light.position.w() == 1 || light.position.w() == 0);
```

Comment:
I hope we're okay with this assert fail if someone uses the python light_setup API incorrectly.

Reply:
I'd print a clearer message here:

```suggestion
  CORRADE_ASSERT(light.position.w() == 1 || light.position.w() == 0,
    "Light position" << light.position << "is expected to have last component either 0 for a directional light or 1 for a point light", {});
```